Extend webhooks table with doc reference

lzap opened the pull request:
SSIA

Cherry-pick into:

* 3.1
* 2.5 (or let me know and I will file PR - I have a customer case)

melcorr commented on guides/common/modules/ref_webhooks-available-events.adoc:
```diff
@@ -4,44 +4,46 @@
 `Action` events trigger webhooks only on `success`, so if an action fails, a webhook is not triggered.
 
+For more information about payload, go to *Administer* > *About* > *Support* > *Templates DSL*.
+Some events are marked as *custom*, in that case payload is an ad-hoc object (key-value data structure) and a list of available keys is provided in the table below.
```
```suggestion
Some events are marked as *custom*, in that case, the payload is an ad-hoc object (key-value data structure) and a list of available keys is provided in the following table:
```

melcorr reviewed (changes requested):
Few small changes and I'll merge and cp!

lzap commented:
Thanks, added but I also added one more hunk in the introduction. I got feedback that we do not explain payload contents too much, so that should do it.

melcorr commented on guides/common/modules/ref_webhooks-available-events.adoc:
```diff
@@ -4,44 +4,46 @@
+|Actions Remote Execution Run Host Job Foreman OpenSCAP Run Scans Succeeded |Run OpenSCAP scan.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Ansible Run Host Succeeded |Runs an Ansible playbook containing all the roles defined for a host.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Ansible Run Capsule Upgrade Succeeded |Upgrade Capsules on given Capsule server hosts.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Ansible Configure Cloud Connector Succeeded |Configure Cloud Connector on given hosts.|Actions::RemoteExecution::RunHostJob
```
This looks like a real mess now on preview

melcorr reviewed (changes requested):
The table no longer makes sense to me at all.

melcorr commented:
Build your table and compare with what exists currently: https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#webhooks-available-events_admin

lzap commented:
Fixed.

melcorr commented on guides/common/Makefile:
```diff
@@ -61,7 +61,7 @@ $(DEST_DIR)/$(BUILD).css: $(BUILD_DIR)/default.css
 $(IMAGES_TS): $(IMAGES)
 	cp $? $(IMAGES_DIR)/
-	pushd "$(IMAGES_DIR)" >/dev/null && for IMG in *.*; do ln -s ../../images/$$IMG ../common/images/$$IMG; done && popd >/dev/null
+	pushd "$(IMAGES_DIR)" >/dev/null && for IMG in *.*; do ln -fs ../../images/$$IMG ../common/images/$$IMG; done && popd >/dev/null
```
@lzap what's happening here?

lzap replied:
Oh that is just adding the force option - in some rare cases Makefile would attempt to copy over existing files and errors out. This change makes it to just overwrite them. It does not do this always, just in edge cases.

melcorr reviewed (approved):
It LGTM now but what is going on with the update to the makefile?

melcorr commented:
5677a79e..ce2dffd8  3.1 -> 3.1
   a4e2abe3..9dcc5591  2.5 -> 2.5

ofedoren commented on guides/common/modules/ref_webhooks-available-events.adoc:
```diff
@@ -4,44 +4,46 @@
+|Actions Remote Execution Run Host Job Leapp Pre-upgrade Succeeded |Upgradeability check for RHEL 7 host.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Leapp Remediation Plan Succeeded |Run Remediation plan with Leapp.|Actions::RemoteExecution::RunHostJob
+|Actions Remote Execution Run Host Job Leapp Upgrade Succeeded |Run Leapp upgrade job for RHEL 7 host.|Actions::RemoteExecution::RunHostJob
+|Build Entered |A host entered the build mode.|Custom event: @payload.id (host id), @payload.hostname (host name).
```
@lzap, I'm so sorry I'm late to the party, but `@payload` is not an object you can call different methods dynamicly. It's a simple `Hash` object. The `@payload` can contain `:object` key, under which is actual Active Record object will be stored, on which you can call those methods. Just for simplicily this same object is available under `@object` variable if present.

So, just to summarize: 
 - `@payload` is a `Hash` object, to retrieve the info from it, one must use e.g. `@palyoad[:hostname]`
 - `@object` is an Active Record object, which can understand its own methods, thus one can call e.g. `name` on it directly: `@object.name`. The same object is available in `@payload[:object]`, so `@object.name` is the same as `@payload[:object].name`.

melcorr replied:
@ofedoren since this is merged, could you open a PR please?

lzap replied:
My bad, I will fix this.

lzap replied:
https://github.com/theforeman/foreman-documentation/pull/951
